exercises/practice/clock/.docs/instructions.append.md

```diff
@@ -7,11 +7,11 @@ your Clock type need to work with the == operator. This means that if your
 New function returns a pointer rather than a value, your clocks will
 probably not work with ==.
 
-While the time.Time type in the standard library (https://golang.org/pkg/time/#Time)
+While the time.Time type in the standard library [https://golang.org/pkg/time/#Time](https://golang.org/pkg/time/#Time)
 doesn't necessarily need to be used as a basis for your Clock type, it might
 help to look at how constructors there (Date and Now) return values rather
 than pointers. Note also how most time.Time methods have value receivers
 rather than pointer receivers.
 
 For some useful guidelines on when to use a value receiver or a pointer
-receiver see: https://github.com/golang/go/wiki/CodeReviewComments#receiver-type
+receiver see: [https://github.com/golang/go/wiki/CodeReviewComments#receiver-type](https://github.com/golang/go/wiki/CodeReviewComments#receiver-type)
```

Comment:
```suggestion
receiver see [Go Wiki: Receiver Type](https://github.com/golang/go/wiki/CodeReviewComments#receiver-type)
```

Whenever possible in the Go track, we try to make the links to blog posts / articles look like `source: title`. Since we are fixing the links, let's also make the links follow this format.